packages/element/examples/view-3d-2d.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Compare</title>
+    <link rel="stylesheet" href="index.css" />
+    <script type="module" src="../src/itk-viewer-element.ts"></script>
+    <script type="module" src="../src/itk-viewport.ts"></script>
+    <script type="module" src="../src/itk-view-2d.ts"></script>
+    <script type="module" src="../src/itk-view-2d-vtkjs.ts"></script>
+    <script type="module" src="../src/itk-view-3d.ts"></script>
+    <script type="module" src="../src/itk-view-3d-vtkjs.ts"></script>
+
+    <script type="module" src="./compare.ts"></script>
+  </head>
+
+  <body>
+    <itk-viewer class="fill" style="flex-direction: row; gap: 0.25rem">
+      <itk-viewport class="fill">
+        <itk-view-3d class="fill">
+          <itk-view-3d-vtkjs></itk-view-3d-vtkjs>
+        </itk-view-3d>
```

Comment:
Can we refactor so this is just:

```html
<itk-view-3d renderer="vtkjs" class="fill"></itk-view-3d>
```

?

Reply:
Good idea.  I'll do it now.